pkgs/servers/misc/podgrab/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "podgrab";
+  version = "2021-04-14"; # No official version tags
+
+  src = fetchFromGitHub {
+    owner = "akhilrex";
+    repo = pname;
+    rev = "3179a875b8b638fb86d0e829d12a9761c1cd7f90";
+    sha256 = "sha256-vhxIm20ZUi+RusrAsSY54tv/D570/oMO5qLz9dNqgqo=";
+  };
+
+  vendorSha256 = "sha256-xY9xNuJhkWPgtqA/FBVIp7GuWOv+3nrz6l3vaZVLlIE=";
+
+  postInstall = ''
+    mkdir -p $out/share/
+    ln -s -r "$src/client" "$out/share/"
+    ln -s -r "$src/webassets" "$out/share/"
+  '';
+
+  meta = with lib; {
+    description = "A self-hosted podcast manager to download episodes as soon as they become live";
+    homepage = "https://github.com/akhilrex/podgrab";
+    license = licenses.gpl3;
+    platforms = platforms.unix;
```

```suggestion
    license = licenses.gpl3;
```

gpl3Ony or gpl3Plus?

platforms is set by buildGoModule to all platforms go works on which is a bit smaller than unix.

---

Ah great, I didn't know about that.

Regarding the license, I'm not sure whether it is gpl3+ or not, I assume it should be mentioned in the license file?